tests/testdata/nonmem/pheno_block.mod

```diff
@@ -1,6 +1,6 @@
 $PROBLEM PHENOBARB SIMPLE MODEL
 $DATA pheno.dta IGNORE=@
-$INPUT ID TIME AMT WGT APGR DV
+$INPUT ID TIME AMT WGT APGR DV FA1
```

Comment:
I need this for `same-as-iiv` tests that implicitly yield a mix of disjoint and joint distributions.